Lib/timeit.py

```diff
@@ -133,7 +133,7 @@ def __init__(self, stmt="pass", setup="pass", timer=default_timer,
         exec(code, global_ns, local_ns)
         self.inner = local_ns["inner"]
 
-    def print_exc(self, file=None):
+    def print_exc(self, file=None, **kwargs):
```

Comment:
This change of signature should be documented I guess. And by default, I wouldn't add colors when timeit is used programatically. I would only add it when being used in the CLI.